pandas/io/parquet.py

```diff
@@ -263,6 +270,11 @@ def read(
 
             if manager == "array":
                 result = result._as_manager("array", copy=False)
+
+            if pa_table.schema.metadata:
+                if b"df.attrs" in pa_table.schema.metadata:
+                    df_metadata = pa_table.schema.metadata[b"df.attrs"]
+                    result.attrs = ast.literal_eval(df_metadata.decode("utf-8"))
```

Comment:
`json.loads`?

Reply:
replaced `ast.literal_eval` to using `json.loads`